pyvkfft/test/test_fft.py

```diff
@@ -412,6 +413,44 @@ def test_pycuda_streams(self):
                     dn = np.roll(d, i * 7, axis=1)
                     self.assertTrue(np.allclose(dn, vd[i].get(), rtol=rtol, atol=abs(dn).max() * rtol))
 
+    @unittest.skipIf(not has_pyopencl, "pyopencl is not available")
+    def test_pyopencl_queues(self):
+        """
+        Test multiple FFT with queues different from the queue used
+        in creating the VkFFTApp
+        """
+        for dtype in (np.complex64, np.complex128):
```

Comment:
You need to check for `complex128`/fp64 support

Reply:
I don't understand. I'm already checking for complex128.